migration/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm missing the copyright notice in this file. You can find the notice template [here](https://github.com/AxonFramework/AxonFramework/blob/master/copyright-template.xml)

Reply:
Applied; surprised there's no automated enforcement (or fix) for missing headers; could take away some of the review burden.

Reply:
Brilliant idea, @timtebeek. I've grown accustomed to spotting it myself, but it's good to offload that concern.